site/js/app.js

```diff
@@ -0,0 +1,72 @@
+
+const plLogo = `
```

Comment:
Hey @apotapov, I have updated the npm package to be used as a js library by unpkg CDN. This can now be updated in `base.html` as follows:

```html
<script src="https://unpkg.com/@xanadu_ai/web-easter-eggs@0.1.10/lib/index.js"></script>
<script>
        window.XanaduEasterEggs.consoleLogEasterEgg('pennylane')
</script>
```